.npmrc

```diff
@@ -1 +1,3 @@
 engine-strict=true
+registry=https://registry.npmjs.org/
```

Comment:
können wir hier nicht die neue registry auf github angeben bzw die alte entfernen?

Reply:
Makes sense.
@FreiLoris could you please revert this line?

Reply:
@thomas-krueger-axa Done!😉

Reply:
@amir-ag Wir müssen hier gar keine angeben. Hat vorhin auch nur dieses engin-strict drin gehabt... Das andere habe ich zum Testen reingemacht. Also die Action läuft auch ohne die registry im npmrc file durch